app/scripts/app.js

```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom'
 import Router from 'react-router'
 import routes from './routes'
 
+require('../styles/app.less')
+
 if (process.env.NODE_ENV !== 'production') {
   window.uiDebug = require('debug')
```

Comment:
this could be just `window.debug` so that it becomes clear that it is just our debugging thing.